app/controllers/admin/users.js

```diff
@@ -0,0 +1,13 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
+  onSessionRoute: computed('session.currentRouteName', function() {
+    let currentRouteName = this.get('session.currentRouteName');
```

Comment:
Instead of session service, use the more reliable router service to get the name

Reply:
@niranjan94 I tried using the router service but had to inject routing service like this:
`routing        : Ember.inject.service('-routing')`
and this failed the travis build